lib/plugin.lazy.js

```diff
@@ -107,3 +99,22 @@ async function loadSentry(ctx, inject) {
   <% } %>
   sentryReadyResolve = undefined
 }
+
+// Temporary inject into Nuxt without using Nuxt's inject method,
+// so we can later inject the final value
+// This method is adapted from the inject method in nuxt/vue-app/template/index.js
+function tempInject({ app }, key, value) {
+  key = '$' + key
+  // Add into app
+  app[key] = value
+  // Add into context
+  if (!app.context[key]) {
+    app.context[key] = value
+  }
```

Comment:
It still looks to me like original `inject` won't be able to override that later:\
https://github.com/nuxt/nuxt.js/blob/6ab65fbf4f2b0d2348363c43f5c2fe81ef47cbc6/packages/vue-app/template/index.js#L177-L179

Reply:
Yeah, will delete that one before injecting again